law.cfg

```
@@ -18,7 +18,7 @@ gfal2: WARNING
 
 [analysis]
 
-default_analysis: mtt.config.analysis_mtt.analysis_mtt
+default_analysis: mtt.config.Run2.analysis_mtt.analysis_mtt
```

I would use lowercase `run2` here for consistency with the other analyses and also since the config is technically a Python module which are typically named using all-lowercase by convention.
```suggestion
default_analysis: mtt.config.run2.analysis_mtt.analysis_mtt
```